configs/webpack-config-compass/.depcheckrc

```diff
@@ -13,5 +12,7 @@ ignores:
   - 'webpack-cli'
   # used implicitly by the webpack loaders file
   - 'browserslist'
+  - '@babel/runtime'
+  - 'babel-plugin-istanbul'
   # recursive
-  - 'mongodb-compass'
\ No newline at end of file
+  - 'mongodb-compass'
```

Comment:
Why is `mongodb-compass` in a lot of the depchecks in Compass? Is it something we can remove or am I missing something? Sorry for bringing up the unrelated, mentioning because the diff made me notice it. I can ask in our slack channel if better than here.

Reply:
A lot of plugins have something similar to `@import (reference) 'mongodb-compass/src/app/styles/index.less'` in their code. Initially all these styles were duplicated in every plugin causing issues when global styles were updated in one place, but not in some other, at some point we just normalized all of it and changed imports to be from `mongodb-compass` instead (here's this PR https://github.com/mongodb-js/compass/pull/2448). At the same time adding them as a dependency to other packages makes everything recursively dependent and even though this is sorta okay in our case and doesn't affect anything, it will confuse the heck out of lerna and it will just refuse to run any commands if we explicitly have this as a dependency in plugins, so for the time being we have this dependency, but ignore with depcheckrc when depchecking

The PR that I linked to have a bit more context on this in the description and discussions

Reply:
Ah thanks, makes sense. Thanks for the pr link.